cmd/xmidt-agent/xmidt_agent.yaml

```diff
@@ -26,22 +26,22 @@ operational_state:
   boot_time: "2024-02-28T01:04:27Z"
 xmidt_service:
   url: ${MAIL}
-# Optional
+# Optional (Examples)
 # storage:
 #   temporary: ~/local-rdk-testing/temporary
 #   durable: ~/local-rdk-testing/durable
 # mock_tr_181:
 #   enabled: true
 #   file_path: xmidt-agent/internal/wrphandlers/mocktr181/mock_tr181.json
-externals:
-  -
-    file: /home/schmidtw/sync/work/oss/xmidt-agent/main/xmidt-agent.main/cmd/xmidt-agent/wes.txt
```

Comment:
i figured we meant to comment this out or remove it 🙂

Reply:
This was added as a mistake & should be here at all :crying_cat_face: